x/observer/keeper/ballot.go

```diff
@@ -30,18 +42,21 @@ func (k Keeper) GetBallot(ctx sdk.Context, index string) (val types.Ballot, foun
 	return val, true
 }
 
-func (k Keeper) GetBallotList(ctx sdk.Context, height int64) (val types.BallotListForHeight, found bool) {
+func (k Keeper) GetBallotListForHeight(ctx sdk.Context, height int64) (val types.BallotListForHeight, found bool) {
```

Comment:
Would just returning a `types.BallotListForHeight` be enough for the caller?

Reply:
I think it should be fine , but this pr just renames the function , 
We can consider changing the signature in a separate PR, as the function is used in a few different places.